src/views/Editor/index.js

```diff
@@ -118,10 +119,11 @@ var EditorPage = React.createClass({
         } else {
             return (
                 <span>
+                    <Loader loaded={!this.props.editor.isSending} scale={1}/>
                     <RaisedButton
                         style={buttonStyle}
                         primary={true}
-                        disabled={this.props.user && !this.props.user.organization}
+                        disabled={this.props.editor.isSending || (this.props.user && !this.props.user.organization)}
                         label={labelText}
```

Comment:
Isn't the button showing a loader? Or is the loader icon diplayed separately?

If the latter, then I suppose we should replace the original text witht "Julkaistaan tapahtumaa...".

(I thought that the publishing times had come down to reasonable lengths.)